### PR TITLE
[#370] Ability to configure ping_interval and ping_max (with tests)

### PR DESCRIPTION
**Pull request by vkhorozvs**
This adds tests to an original code review here: https://github.com/nats-io/gnatsd/pull/187

### CODE REVIEW

**Comment by vkhorozvs** [comment]
All tests passed, a failing static check has nothing to do with my commit:

~~~
$ staticcheck ./...
/home/travis/gopath/src/github.com/nats-io/gnatsd/test/monitor_test.go:543:2: this value of send is never used
/home/travis/gopath/src/github.com/nats-io/gnatsd/server/monitor_test.go:1311:3: the goroutine calls T.Fatalf, which must be called in the same goroutine as the test
/home/travis/gopath/src/github.com/nats-io/gnatsd/server/monitor_test.go:1311:3: the goroutine calls T.Fatalf, which must be called in the same goroutine as the test
/home/travis/gopath/src/github.com/nats-io/gnatsd/server/monitor_test.go:1311:3: the goroutine calls T.Fatalf, which must be called in the same goroutine as the test
/home/travis/gopath/src/github.com/nats-io/gnatsd/server/monitor_test.go:1311:3: the goroutine calls T.Fatalf, which must be called in the same goroutine as the test
The command "staticcheck ./..." exited with 1.
~~~

**Comment by kozlovic** [comment]
@vkhorozvs Once https://github.com/nats-io/gnatsd/pull/374 is merged, you could rebase with master and it is likely that the test will then pass.

**Comment by vkhorozvs** [comment]
@kozlovic done, thank you!

File: server/configs/test.conf [hunk 1]
```diff
@@ -36,3 +36,7 @@ max_payload: 65536
 
 # slow consumer threshold
 max_pending_size: 10000000
+
+# ping interval and no pong threshold
```
Comment (kozlovic) [inline server/configs/test.conf]: By "no pong" you mean "number of"?

Comment (vkhorozvs) [inline server/configs/test.conf]: I meant "number of pings without a pong (pings with no pong)".
Please, let me know if you think there is a better yet brief description.

Comment (kozlovic) [inline server/configs/test.conf]: I see. No it's fine with me.

**Comment by kozlovic** [comment]
LGTM

**Comment by kozlovic** [comment]
@derekcollison This PR from @vkhorozvs adds ability to configure ping interval and max pings. I have reviewed and looks good to me. The configuration converts entries in config as seconds, which I think is reasonable (we use much lower values in tests though, but through Options directly so that's fine).

**Pull request merged**